src/types/drupal/field_type.d.ts

```diff
@@ -102,3 +104,19 @@ export interface BreadCrumbLink {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   options?: any[]
 }
+
+/**
+ * Types for services and health services
+ */
+
+export interface HealthService {
```

Comment:
`FieldHealthService` to match the other field types in this file and be a little clearer between this & the formatted type

Reply:
Fixed